### PR TITLE
change-message-format-check

### PR DESCRIPTION
**Pull request by BoobeshPalanisamy**

### CODE REVIEW

File: config/whatsapp.js [hunk 1]
```diff
@@ -7,13 +7,21 @@ const restAPI = whatsAppClient.restAPI({
...
+    *Phone:* ${createdEnquiry.phoneNumber}
+    *Email:* ${createdEnquiry.email}
+  
+    *✉️Message:*
+    ${createdEnquiry.message}
+    `;
 
     const result = await restAPI.message.sendMessage(
-      "916382440493@c.us",
+      "919047760131@c.us",
```
Comment (praveensumathi) [inline config/whatsapp.js:24]: get it from the ENV file

File: config/whatsapp.js [hunk 2]
```diff
@@ -7,13 +7,21 @@ const restAPI = whatsAppClient.restAPI({
...
 const sendMessageToWhatsapp = async (createdEnquiry) => {
   try {
-    const message = `Hey Hi, You got an enquiry, The child name is ${createdEnquiry.childName} 
-      and the Parentname is ${createdEnquiry.parentName}. 
-      Contact info Phone: ${createdEnquiry.phoneNumber} and Email: 
-      ${createdEnquiry.email} along with the message ${createdEnquiry.message}.`;
+    const message = `
+    *👋 Hi,* You got an enquiry:
+
+    *🧒Child Name:* ${createdEnquiry.childName}
```
Comment (praveensumathi) [inline config/whatsapp.js:13]: check the empty names, child name and parent name

File: config/whatsapp.js (hunk 1 above)
Comment (BoobeshPalanisamy) [reply config/whatsapp.js:24]: moved to env and receiving value

Comment (BoobeshPalanisamy) [reply config/whatsapp.js:24]: Cant move the package validate at the particular place should be in proper format

error message:
message: "Validation failed. Details: 'chatId' must be one of the next formats: 'phone_number@c.us' or 'group_id@g.us'"

File: config/whatsapp.js (hunk 2 above)
Comment (BoobeshPalanisamy) [reply config/whatsapp.js:13]: checked

**Review by praveensumathi — APPROVED** [review]

**Pull request merged**